test/python/basicaer/test_multi_registers_convention.py

```diff
@@ -39,17 +39,17 @@ def test_circuit_multi(self):
 
         backend_sim = BasicAer.get_backend("qasm_simulator")
 
-        result = execute(qc, backend_sim, seed_transpiler=34342).result()
+        result = backend_sim.run(qc, seed_transpiler=34342).result()
         counts = result.get_counts(qc)
 
         target = {"01 10": 1024}
 
         backend_sim = BasicAer.get_backend("statevector_simulator")
-        result = execute(circ, backend_sim, seed_transpiler=3438).result()
+        result = backend_sim.run(circ, seed_transpiler=3438).result()
         state = result.get_statevector(circ)
 
         backend_sim = BasicAer.get_backend("unitary_simulator")
-        result = execute(circ, backend_sim, seed_transpiler=3438).result()
+        result = backend_sim.run(circ, seed_transpiler=3438).result()
```

Comment:
These `seed_transpiler` kwargs look spurious now that the `transpile` call has been elided.

Reply:
oops, my bad. Removing it in 05dd6869a88ad51e42e8ae682e82f6fa06e837fd